src/profile-logic/profile-data.js

```diff
@@ -2049,6 +2049,26 @@ export function findBestAncestorCallNode(
   return clickedCallNode;
 }
 
+/**
+ * Return the root call node of the selected call node.
+ */
+export function findRootCallNode(
+  selectedCallNode: IndexIntoCallNodeTable,
```

Comment:
nit: this isn't always the `selected` call node here, this is any call node. 

But actually this new function might not be necessary, see the other comment above. I think you can just remove it if you follow my suggestion above :-)

Reply:
I followed your suggestion and removed the function, as it wasn't needed anymore.